.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files

```

Comment:
I guess these couple of folders we don't need to push?

Reply:
not needed yes